canonical-websites/www.ubuntu.com
# 3881: Merge beta branch into master

## Done

- Merged beta branch into master branch
- Fixed up some styling/templating errors on the /engage (English + German) pages
- Temporarily removed `/search` from Travis linkchecker
- Updated Rigado webinar CTA to say "Watch on-demand"
- Small styling fixes

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Check the primary nav, secondary nav and global nav work as expected
- Check the following new pages:
  - /openstack + children
  - /kubernetes + children
  - /ai + children
  - /about + children
  - /containers
  - /engage/vmware-to-openstack + /engage/de/vmware-to-openstack
  - /public-cloud

## Issue / Card

Fixes #3881, fixes ubuntudesign/web-squad#764
